pkg/templates/templates.go

```diff
@@ -0,0 +1,200 @@
+// Copyright (c) 2020 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
```

Comment:
Sorry, I should have been clearer (and the copyright thing wasn't really clear to anyone, so that may be part of it). The original copyright header should remain in the file, and we can add ours below it (like the OCM line that was added at the bottom there)--basically if there's already a header there we don't touch it, but we can add our  "Copyright Contributors to the Open Cluster Management project" and optionally a "Copyright (c) 2021 Red Hat, Inc." below the header that's already present.